Frontend/src/components/Home.jsx

```diff
@@ -2,6 +2,8 @@ import React from 'react';
 import { motion } from 'framer-motion';
 import { Link } from 'react-router-dom';
 import { FaHospital, FaUserMd, FaCalendarCheck, FaHeartbeat, FaAmbulance, FaLaptopMedical } from 'react-icons/fa';
+import Support from './Support';
```

Comment:
Change the style of the button. Add a support icon and position it on the right side just above the scroll to top button.

Reply:
<img width="158" alt="Screenshot 2024-10-30 at 5 35 04 AM" src="https://github.com/user-attachments/assets/10ecda8b-555f-4416-b305-6d79cb47bb47">